src/net64/emulator/m64plus.hpp

```diff
@@ -32,7 +32,7 @@ namespace M64PTypes
 namespace M64PlusHelper
 {
 
-struct Plugin;
+class Plugin;
```

Comment:
`Plugin` is now private and friends with `Mupen64Plus` and since you used the default visibility of `struct` I'm using the default visibility of `class`